forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabRepository.java

```diff
@@ -828,4 +828,19 @@ public void deleteLabel(Label label) {
         request.delete("labels/" + label.name())
                 .execute();
     }
+
+    @Override
+    public int deleteDeployKeys(Duration duration) {
+        var expiredKeys = request.get("deploy_keys").execute()
+                .stream()
+                .filter(key -> ZonedDateTime.parse(key.get("created_at").asString())
+                        .isBefore(ZonedDateTime.now().minusSeconds(duration.toSeconds())))
```

Comment:
No need to convert to seconds when using `Duration`. It implements `TemporalAmount` so you can just:
```suggestion
                        .isBefore(ZonedDateTime.now().minus(duration)))
```

Reply:
Good to know. Will fix it too.